account_members_test.go

```diff
@@ -215,8 +252,12 @@ func TestCreateAccountMemberWithStatus(t *testing.T) {
 	}
 
 	mux.HandleFunc("/accounts/01a7362d577a6c3019a474fd6f485823/members", handler)
+	accountResource := &ResourceContainer{
```

Comment:
from the code docs - https://github.com/cloudflare/cloudflare-go/blob/master/resource.go#L12-L14

```
// ResourceContainer defines an API resource you wish to target. Should not be
// used directly, use `UserIdentifier`, `ZoneIdentifier` and `AccountIdentifier`
// instead.
```

while you can do this, i'd recommend using the helpers instead incase we expand this struct. it's also a bit nicer to read 😄